### DISCUSSION TITLE
消失的休憩 | /cn/2019/08/restless/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1397] [Comments]
https://yihui.org/cn/2019/08/restless/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
娃还太小，确实没有进入游玩状态。他们这个阶段其实不在乎在哪里玩，去爬落基山并不一定比爬家附近的一个土包更开心。对他们而言，也许最开心的就是在山顶餐馆里吃了一顿垃圾食品，至于山水多宏伟壮丽，可能与他们无关。

> ——原帖发布于 2020-08-20 04:36:55

**Comment by ZaiyanHere** [comment]
新晋ap恰好前不久在Denver开了会，一个秘密的计划是——投出去五篇paper之后就一个人开着车去科罗拉多转一圈儿！

**Reply by yihui** [reply]
恭喜晋升！这个秘密计划很好！
